app/models/user.rb

```diff
@@ -0,0 +1,8 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: :author_id
+  has_many :comments, foreign_key: :author_id
+  has_many :likes, foreign_key: 'user_id'
```

Comment:
- [ ] Dear @Tracy-miranja, You did well implementing the required changes, one thing to be incorporated into your changes here is the foreign for `has_many :likes` expected to be `:author_id` instead of `user_id`.